strings/tips.txt

```diff
@@ -148,6 +149,8 @@ As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic see
 As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
 As a Botanist, you can combine production trait chemicals just like a Chemist. Chlorine (blumpkin) + radium and phosphorus (glowshrooms) equals unstable mutagen!
+As a Botanist, Rnd might pester you for different type of seed packets, perhaps a deal can be made in turn.
```

Comment:
Remove, useless tip.

Reply:
How is it useless

Reply:
It's useless, it's not a tip, just a memo of an unlikely roleplaying situation.